tools/ci/launch_cluster.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
We could call this from the makefile?

Reply:
Good point. Looks like `test.sh` calls `test_runner.sh`, which then calls this if `CLUSTER_URL` is unspecified. Switched to using this instead of having a lot of custom cluster launch logic in the spark Makefile.